zstash/create.py

```diff
@@ -54,11 +55,13 @@ def create():
     if hpss != "none":
         url = urlparse(hpss)
         if url.scheme == "globus":
+            # identify globus endpoints
+            logger.debug(f"{ts_utc()}:Calling globus_activate(hpss)")
             globus_activate(hpss)
         else:
             # config.hpss is not "none", so we need to
             # create target HPSS directory
-            logger.debug("Creating target HPSS directory")
+            logger.debug(f"{ts_utc()}:Creating target HPSS directory {hpss}")
```

Comment:
Add space after `:` for consistency.

Reply:
@forsyth2 I expect to make many edits such as this before full acceptance.  My method is to make the changes in my local (chrysalis) repo and commit/push them, rather than make the changes in the web/ui as that confuses me.  For instance, I added "DIVING" and "SURFACE" as log-terms just before and after a major function call, so I could be sure when and where processing was occurring, and I expect to remove those terms.

Reply:
> My method is to make the changes in my local (chrysalis) repo and commit/push them, rather than make the changes in the web/ui 

Agreed, I like that method better too.